tests/framework/utils_iperf.py

```diff
@@ -0,0 +1,132 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""File containing utility methods for iperf-based performance tests"""
+
+import concurrent.futures
+import json
+import time
+
+from framework import utils
+from framework.utils import CmdBuilder, CpuMap, get_cpu_percent, summarize_cpu_percent
+
+DURATION = "duration"
+IPERF3_END_RESULTS_TAG = "end"
+THROUGHPUT = "throughput"
+CPU_UTILIZATION_VMM = "cpu_utilization_vmm"
+CPU_UTILIZATION_VCPUS_TOTAL = "cpu_utilization_vcpus_total"
+
+# Dictionary mapping modes (guest-to-host, host-to-guest, bidirectional) to arguments passed to the iperf3 clients spawned
```

Comment:
nit: Reordering modes in comment looks more prettier:
```suggestion
# Dictionary mapping modes (bidirectional, guest-to-host, host-to-guest) to arguments passed to the iperf3 clients spawned
```

Reply:
mh, I prefer having bidirectional last because that way you get know about what the two directions are first, before learning that we also support doing both at the same time.

Reply:
I see. Then, how about changing `MODE_MAP`? It's a matter of taste, though.
```
MODE_MAP = {"g2h": [""], "h2g": ["-R"], "bd": ["", "-R"]}
```

Reply:
oh, yeah, matching they should